src/ORM/DataObject.php

```diff
@@ -379,12 +380,6 @@ public function __construct($record = null, $isSingleton = false, $queryParams =
 
         $this->original = $this->record;
 
-        // Keep track of the modification date of all the data sourced to make this page
-        // From this we create a Last-Modified HTTP header
-        if (isset($record['LastEdited'])) {
-            HTTP::register_modification_date($record['LastEdited']);
```

Comment:
I'm going to mention the removal of this feature in the changelog. Don't think we need to retain it, but it's noteworthy

Reply:
Adding:

```
* DataObject no longer automatically invokes `HTTP::register_modification_date`
   with LastModified on construction. User code should instead invoke
   `HTTPCacheControlMiddleware::singleton()->registerModificationDate()` explicitly.
```